packages/backend/schema.gql

```diff
@@ -127,7 +127,7 @@ type CommentConnection {
 }
 
 type CommentEdge {
-  cursor: String!
+  cursor: String
```

Comment:
How did this change happen?

Reply:
This was probably done when I was testing, but I'm not sure why it didn't revert back.